File: bubble.py
def printArray(arr):
    for i in range(len(arr)):
        print(arr[i], end=" ")
    print()

arr = [7, 84, 23, 56, 10]

# Bubble sort
for i in range(len(arr) - 1):
    for j in range(len(arr) - i - 1):
        if arr[j] > arr[j + 1]:
            # Swapping
            arr[j], arr[j + 1] = arr[j + 1], arr[j]

printArray(arr)

File: bubble.js
function printArray(arr) {
    for (let i = 0; i < arr.length; i++) {
        process.stdout.write(arr[i] + " ");
    }
    process.stdout.write("\n");
}

let arr = [7, 84, 23, 56, 10];

// Bubble sort
for (let i = 0; i < arr.length - 1; i++) {
    for (let j = 0; j < arr.length - i - 1; j++) {
        if (arr[j] > arr[j + 1]) {
            // Swapping
            let temp = arr[j];
            arr[j] = arr[j + 1];
            arr[j + 1] = temp;
        }
    }
}

printArray(arr);
